### DISCUSSION TITLE
Ugc android discontinued?

### DISCUSSION THREAD

**Discussion by GreenMystic** [Eloston/ungoogled-chromium#2671] [Q&A] [answered]
hi i saw you forked ugc 
https://github.com/ungoogled-software/ungoogled-chromium-android

but it's seems like outdated 
(last release is 2022) 
is this android build discontinued? 🙏

**Comment by networkException** [comment, marked as answer]
Yes, the maintainer is not working on it anymore

**Reply by GreenMystic** [reply]
is there any active fork or similar chromium based browser for android?

ungoogled chromium is plain browser
(i mean without any extra feature)

extra feature can disabled but I won't use them bcs i care about storage rhese feature is obviously increasing app size

**Reply by rmdhnreza** [reply]
@GreenArchiveForever Try [Cromite](https://github.com/uazo/cromite)

**Reply by GreenMystic** [reply]
> @GreenArchiveForever Try [Cromite](https://github.com/uazo/cromite)

sorry i mean without any additional feature like AdBlock
ugc is perfect but unfortunately android build is discontinued :-(
